src/components/button-toggle-group/button-toggle-group.stories.mdx

```diff
@@ -109,6 +109,36 @@ import ButtonToggleGroup from "carbon-react/lib/components/button-toggle-group";
   </Story>
 </Canvas>
 
+### With fullWidth
+
+When the `fullWidth` prop is `true`, the css property flex: auto is applied to each `Button`s 
```

Comment:
```suggestion
When the `fullWidth` prop is `true`, the css property `flex: auto` is applied to each `Button`s 
```

If we keep this wording this CSS string should be rendered with code formatting I think.